olive/evaluator/olive_evaluator.py

```diff
@@ -482,7 +483,7 @@ def _inference(
         is_single_tensor_output = len(output_names) == 1
         for input_data, labels in dataloader:
             input_feed = OnnxEvaluator.format_input(input_data, io_config)
-            result = session.run(input_feed)
+            result = model.run_session(session, input_feed, **run_kwargs)
```

Comment:
this one should still be `session.run`. Here `session` is actually an `OrtInferenceSession` wrapper over `ort.InferenceSession`

Reply:
could rename `session` to `session_wrapper` to avoid the confusion.

Reply:
actually nevermind. model.run_session for onnx model is just a wrapper over the session.run call.